cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -517,6 +643,9 @@ objects:
             readOnly: true
           - name: azure-runtime-config
             mountPath: /configs/azure-runtime-config
+          - name: azure-operators-managed-identities-config
+            mountPath: /configs/azure-operators-managed-identities-config.yaml
+            subPath: azure-operators-managed-identities-config.yaml
```

Comment:
From https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath, I see this NOTE
```
A container using a ConfigMap as a [subPath](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath) volume mount will not receive ConfigMap updates.
```

This means that whenever this config map is changed, cs will need to be redeployed. This is fine for now  as CS doesn't autoreload the file when it changes, but mentioning the note for consideration in case that implementation changes in CS.